public/apps/configuration/panels/permission-list.tsx

```diff
@@ -0,0 +1,242 @@
+    render: renderCustomization,
+  },
+];
+
+export function PermissionList(props: AppDependencies) {
+  const [actionGroups, setActionGroups] = useState<ActionGroupListingItem[]>([]);
+  const [errorFlag, setErrorFlag] = useState(false);
+  const [selection, setSelection] = useState<ActionGroupListingItem[]>([]);
+  const [isActionsPopoverOpen, setActionsPopoverOpen] = useState(false);
+  const [isCreateActionGroupPopoverOpen, setCreateActionGroupPopoverOpen] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setActionGroups(await fetchActionGroupListing(props.coreStart.http));
+      } catch (e) {
+        console.log(e);
+        setErrorFlag(true);
+      }
+    };
+
+    fetchData();
+  }, [props.coreStart.http]);
+
+  const handleDelete = async () => {
+    const usersToDelete: string[] = selection.map((r) => r.name);
+    try {
+      await requestDeleteUsers(props.coreStart.http, usersToDelete);
+      setActionGroups(difference(actionGroups, selection));
+      setSelection([]);
+    } catch (e) {
+      console.log(e);
+    } finally {
+      setActionsPopoverOpen(false);
+    }
+  };
+
+  const actionsMenuItems = [
+    <EuiContextMenuItem
+      key="edit"
+      // TODO edit action
+      onClick={() => {}}
+      disabled={selection.length !== 1}
+    >
+      Edit
+    </EuiContextMenuItem>,
+    <EuiContextMenuItem
+      key="duplicate"
+      // TODO duplicate action
+      onClick={() => {}}
+      disabled={selection.length !== 1}
+    >
+      Duplicate
+    </EuiContextMenuItem>,
+    <EuiContextMenuItem key="delete" onClick={handleDelete} disabled={selection.length === 0}>
+      Delete
+    </EuiContextMenuItem>,
+  ];
+
+  const createActionGroupMenuItems = [
+    <EuiContextMenuItem
+      key="blank"
+      // TODO create empty
+      onClick={() => {}}
+    >
+      Create from blank
+    </EuiContextMenuItem>,
+    <EuiContextMenuItem
+      key="blank"
+      // TODO create from selection
+      onClick={() => {}}
+      disabled={selection.length === 0}
+    >
+      Create from selection
+    </EuiContextMenuItem>,
+  ];
+
+  const actionsButton = (
+    <EuiButton
+      iconType="arrowDown"
+      iconSide="right"
+      onClick={() => {
+        setActionsPopoverOpen((prevState) => !prevState);
+      }}
+    >
+      Actions
+    </EuiButton>
+  );
+
+  const createActionGroupMenuButton = (
+    <EuiButton
+      iconType="arrowDown"
+      iconSide="right"
+      onClick={() => {
+        setCreateActionGroupPopoverOpen((prevState) => !prevState);
+      }}
+      fill
+    >
+      Create action group
+    </EuiButton>
+  );
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>Permission</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle size="s">
+              <h3>Permissions ({actionGroups.length})</h3>
```

Comment:
Looks like the number has a different font style from mockup.

Reply:
I believe there was a discussion in previous PR for other listing page. The EuiTitle controls the bold style.